client/allocrunner/alloc_runner_hooks.go

```diff
@@ -138,6 +138,7 @@ func (ar *allocRunner) initRunnerHooks(config *clientconfig.Config) error {
 	alloc := ar.Alloc()
 	ar.runnerHooks = []interfaces.RunnerHook{
 		newAllocDirHook(hookLogger, ar.allocDir),
+		&cgroupHook{alloc: ar.Alloc(), cpusetManager: ar.cpusetManager},
```

Comment:
Even though its currently pretty simple it seems like an `new` func here might be the way to go